tools/scripts/rewrite-awx-operator-requirements.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import yaml
+import sys
+
+
+AWX_ENTRY = 'https://github.com/ansible/awx.git#/awx_collection/'
+
+
+def load_yaml_file(fname):
+    with open(fname, 'r') as file:
+        data = yaml.safe_load(file)
+    return data
+
+
+def write_yaml_file(fname, data):
+    with open(fname, 'w') as file:
+        yaml.dump(data, file)
+
+
+def replace_awx(data, path):
+    for entry in data['collections']:
+        if entry['name'] == AWX_ENTRY or entry['name'] == 'awx.awx':
```

Comment:
Code is written to work with or without awx-operator PR https://github.com/ansible/awx-operator/pull/1990